olive/cli/run.py

```diff
@@ -23,6 +23,7 @@ def register_subcommand(parser: ArgumentParser):
         )
         sub_parser.add_argument("--run-config", "--config", type=str, help="Path to json config file", required=True)
         sub_parser.add_argument("--setup", help="Whether run environment setup", action="store_true")
```

Comment:
can you make setup and package part of a mutually exclusive group? 
```python
dependency_group = sub_parser.add_mutually_exclusive_group()
dependency_group.add_argument("--setup", action="store_true")
dependency_group.add_argument("--packages", action="store_true")
```

Reply:
`packages` is only mutually exclusive with engine run. If have both `setup` and `packages`, Olive will generate requirements packages and install them both, in case user wants to know which packages installed as well.

Reply:
![image](https://github.com/user-attachments/assets/af24d27d-bf66-4450-9ad3-8d2a61ffe424)
the logic here makes them almost mutually exclusive since it exists after the first one

Reply:
if both can be true together, the return needs to be outside the nested ifs

Reply:
Ah yeah true. I forgot to update this. Updated.